maybe-build.sh

```diff
@@ -2,9 +2,9 @@
 # Build
 mkdir -p "${GITHUB_WORKSPACE}/${BUILD_DIR}"
 cd "${GITHUB_WORKSPACE}/${BUILD_DIR}";
-if [ -f "$BUILD_DIR/.github/build-for-deployment.sh" ]; then
```

Comment:
I'd rather fix it once and for all and have the full path on both occasions, removing the cd in line 4.